x-pack/plugins/observability_solution/apm/public/tutorial/config_agent/opentelemetry_instructions.tsx

```diff
@@ -110,7 +110,10 @@ export function OpenTelemetryInstructions({ apmServerUrl, secretToken }: Props)
                 target="_blank"
                 href="https://github.com/open-telemetry/opentelemetry-specification/blob/v1.10.0/specification/protocol/exporter.md"
               >
-                OTEL_EXPORTER_OTLP_ENDPOINT
+                {i18n.translate(
+                  'xpack.apm.openTelemetryInstructions.otelexporterotlpendpointLinkLabel',
+                  { defaultMessage: 'OTEL_EXPORTER_OTLP_ENDPOINT' }
+                )}
```

Comment:
Here as well. 

cc: @achyutjhunjhunwala

Reply:
Eh I ran the `eslint --fix` locally and it auto fixed hardcoded strings into those, i will just checkout these files from `main`

Reply:
I reverted all the changes from the linter, can you have another look please?